pkg/kubeapiserver/authorizer/config.go

```diff
@@ -75,7 +75,7 @@ func (config AuthorizationConfig) New() (authorizer.Authorizer, authorizer.RuleR
 			node.AddGraphEventHandlers(
 				graph,
 				config.InformerFactory.Core().InternalVersion().Nodes(),
-				config.InformerFactory.Core().InternalVersion().Pods(),
+				config.VersionedInformerFactory.Core().V1().Pods(),
```

Comment:
we should only externalize pod informer here, am i right?

Reply:
Moving one resource type at a time would minimize the size of the PR

Reply:
> Moving one resource type at a time would minimize the size of the PR

make sense 👍